x-pack/plugins/osquery/cypress/tasks/live_query.ts

```diff
@@ -117,8 +117,9 @@ export const toggleRuleOffAndOn = (ruleName: string) => {
 };
 
 export const loadRuleAlerts = (ruleName: string) => {
-  cy.login(ServerlessRoleName.SOC_MANAGER);
```

Comment:
I *think* this removal is causing the cypress test failures. All the tests are stuck on a login screen and the failing tests are all using `loadRuleAlerts`.

Reply:
reverted!